openerp/addons/base/migrations/9.0.1.3/noupdate_changes.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='utf-8'?>
+<openerp>
+    <data>
+        <record model="res.groups" id="group_user">
+            <field name="implied_ids" eval="[(4, ref('group_no_one'))]"/>
```

Comment:
@elicoidal If you refer to this question by @hbrunn, odoo 9 assigns group_no_one to everyone, as you can see here: https://github.com/odoo/odoo/blob/9.0/openerp/addons/base/security/base_security.xml#L45

So all users have access to the group "Technical Features" in Odoo9 standard, by default. It's just that they have to activate the debugging mode in order to see them. I am not introducing any new element in the migration, just forcing that the group "group_user" behaves as it should on a migrated v9, as it would work for a plain vanilla v9.